firefox-ios/Client/Frontend/Home/Homepage Rebuild/HomepageViewController.swift

```diff
@@ -596,6 +602,16 @@ final class HomepageViewController: UIViewController,
     }
 
     // MARK: Dispatch Actions
+    private func recalculateTopSitesDisplay(with width: CGFloat) {
+        currentNumberOfTilesPerRow = numberOfTilesPerRow(for: width)
+        store.dispatch(
+            HomepageAction(
+                windowUUID: windowUUID,
+                actionType: HomepageActionType.viewWillTransitionCalled
+            )
+        )
```

Comment:
It feels odd to me to not have the number of tiles as payload associated with the dispatch action. But I kept the changes simple instead of passing the int around. Let me know what you think!